### DISCUSSION TITLE
Repeater-Field possible?

### DISCUSSION THREAD

**Discussion by impactfactory** [pocketbase/pocketbase#1536] [Q&A] [answered]
Could be very convenient for an editor to have "[Repeater-Field](https://www.google.com/search?q=octobercms+repeater+field&source=lnms&tbm=isch&sa=X&ved=2ahUKEwj_qpbyr7j8AhXf8rsIHfl2DzUQ_AUoAnoECAEQBA&biw=2116&bih=1967&dpr=1#imgrc=Tz7NSB_TFoKKJM)"-Typ as f.e. OctoberCMS/WinterCMS have. Its basically a field, where you can add a programmed, repetitive set of other fields. same structure as a json-field but fastly clickable. with an accordoen its provides a nice overview over the items. Seems for me it'sa more convenient solution instead of making a releation field when the entries are not repetitive at all.

Something like this, is getting uneditable in pocketbase (and I don't understand how to deal with the image, I need):

`{[
    {"title": "Find a place",
     "img": "theoptimalplace.jpg",
     "desc":"look out for fitting place",
     "substeps": [
            "check sunlight",
            "check soil"
            "check temperature",
     ]
    },
    {"title": "Grube ausheben",
     "img": "pic.jpg",
     "desc":"macht man so und so",
     "substeps": [
            "mach das erste",
            "mach das zweite"
            "mach das dritte",
     ]
    },
    {"title": "Badwanne positionieren",
     "img": "pic.jpg",
     "desc":"macht man so und so",
     "substeps": [
            "mach das erste",
            "mach das zweite"
            "mach das dritte",
     ]
    }
]
}`

With a repeater field where I could config that the fields title, img, desc and substeps are repeated, I only had to deal in substeps with an json syntax.

Is there already a solution to such a need I do not understand?

**Comment by ganigeorgiev** [comment, marked as answer]
"Nested collections/Repeatable fields/Field groups" are in my todo to research them but it is a very low priority and for now it is left out of the scope for [v1.0.0](https://github.com/orgs/pocketbase/projects/2).

For now the best way to handle structured relational data is to use a separate collection and link it via a `relation` field.

_Note that the next v0.12 release will be focused on Admin UI improvements, including redesign and refactoring of the relation picker as part of #976._

**Reply by impactfactory** [reply]
Thanx for fast as speed answer!

**Reply by nicodigital** [reply]
I loved this manager, I would only be missing this functionality to hire a license. I remain attentive to this modification.
